app/models/application_record.rb

```diff
@@ -8,7 +8,11 @@ def self.fuzzy_search(term, **cols)
 
   def self.match_search(term, **cols)
     sanitized = sanitize_for_search term, **cols
-    select(Arel.sql("`#{table_name}`.*, #{sanitized} AS search_score")).where(sanitized)
+
+    mappedCols = sanitized.map { |val| "#{val} AS search_score_#{sanitized.find_index(val)}" }.join(', ')
+    whereClause = sanitized.map { |val| val.to_s }.join(' OR')
```

Comment:
Style: Ruby variables are named in `snake_case`, i.e. `mapped_cols` and `where_clause`